src/components/Common/UserSelector.tsx

```
@@ -192,32 +199,33 @@ export default function UserSelector({
-        </div>
-      ) : (
-        <span>{placeholder || t("select_user")}</span>
-      )}
-      <CaretDownIcon className="ml-auto" />
-    </Button>
-  );
+  const renderTriggerButton = () =>
+    trigger || (
+      <Button
+        variant="outline"
+        role="combobox"
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<select>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)